Examples/IntelliTect.TestTools.Selenate.Examples/Global.cs

```diff
@@ -0,0 +1,4 @@
+﻿global using IntelliTect.TestTools.Selenate.Examples.Pages;
```

Comment:
Not suggesting a change, but wanted to make you aware of it. These global using statements can be controlled with the .csproj as well. This has the advantage of being able to leverage things like Directory.Build.props files to propagate them to multiple projects. However, I have been told by multiple people they find this un-intuitive.

Reply:
I was aware, yes, but wasn't sure what the best practice was. If you have no strong objections I'll leave this as-is and look to change in the next update (which hopefully won't take nearly as long. :) )